vaas/vaas/adminext/templates/forms/rewrite_groups.html

```diff
@@ -0,0 +1,75 @@
+{% spaceless %}
+<div class="input-group" id="input-group">
+    {% for widget in widget.subwidgets %}
+    {% if forloop.first %}
+    <span class="input-group-addon">
+        {% include widget.template_name %}
+    </span>
+    {% else %}
+    {% include widget.template_name %}
+    {% endif %}
+    {% endfor %}
+</div>
+{% endspaceless %}
+
+<script>
+    (function ($) {
+
+        const Validation = {
+            Unknown: 'unknown',
+            Success: 'has-success',
+            Error: 'has-error',
+        }
+
+        const state = {
+            regex_compilation: 'unknown'
+        };
+
+        function updateTarget(oldState, newState) {
+            $("#input-group").removeClass(oldState)
+            if (newState != Validation.Unknown) {
+                $("#input-group").addClass(newState)
+            }
+        }
+
+        function setRegexCompilationState(newState) {
+            if (state.validation_state != newState) {
+                let old = state.regex_compilation;
+                state.regex_compilation = newState;
+                updateTarget(old, newState);
+            }
+        };
+
+        function regex_dynamic_check() {
```

Comment:
pls use consistent naming convention checkDynamicRegex - or something like that :)